extensions/wikia/PhalanxII/classes/Phalanx.class.php

```diff
@@ -37,8 +35,6 @@ class Phalanx extends WikiaModel implements ArrayAccess {
 		2   => 'summary',
 		4   => 'title',
 		8   => 'user',
-		16  => 'question_title',
-		32  => 'recent_questions',
```

Comment:
Let's clean this up from phalanx service as well.

Reply:
https://github.com/Wikia/scala-backend/pull/114